### ISSUE TITLE
Buttons failing color contrast WCAG AA

### ISSUE THREAD

**Issue by kevfuzz** [primefaces/primeng#19089]
### Describe the bug

Hi,

I notice if you run Lighthouse over the buttons page (https://primeng.org/button), most of the buttons are failing the color contrast check.

Given you are aiming for WCAG AA compliance, is there any plan to adjust the colours to fix this? If not, then what is the recommended approach to making them accessible?

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [x] Other

### Other Reason

Issue is on the PrimeNG showcase (https://primeng.org/button)

### Reproducer

https://stackblitz.com/edit/bagiu96n?file=package.json

### Environment

https://primeng.org/button

### Angular version

20.0.0

### PrimeNG version

v20

### Browser(s)

Chrome

### Steps to reproduce the behavior

Open https://primeng.org/button and run Lighthouse accessibility tests over the page.

### Expected behavior

Buttons should pass color contrast checks.
